src/content/docs/implementation/setup/content-delivery-network.mdx

```diff
@@ -93,9 +93,21 @@ The Fastly configuration instructions on this page are based on the [`fastly-mag
 
    For example: `main--aem-boilerplate-commerce--hlxsites.aem.live`
 
-1. Click **Attach condition**.
+1. Click **Attach condition*.
 
-1. Select `false (REQUEST) false` from the **Condition** drop-down list.
+1. Click **Create a new request condition**.
+
+1. Enter a name for the condition.
+
+   For example: `false`.
+
+1. Enter `false` in the **Apply if** field.
+
+1. Enter `10` in the **Priority** field.
```

Comment:
This is default, you don't need to call it out, or you can say "leave as is". In my screenshot I gave I had just entered false/false.